integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/dialogs/CoinBaseResultDialog.kt

```diff
@@ -33,11 +33,11 @@ import androidx.fragment.app.DialogFragment
 import androidx.navigation.fragment.findNavController
 import org.dash.wallet.common.ui.viewBinding
 import org.dash.wallet.integration.coinbase_integration.R
-import org.dash.wallet.integration.coinbase_integration.databinding.DialogCoinbaseBuyDashBinding
+import org.dash.wallet.integration.coinbase_integration.databinding.DialogCoinbaseResultBinding
 
-class CoinBaseBuyDashDialog : DialogFragment() {
-    private val binding by viewBinding(DialogCoinbaseBuyDashBinding::bind)
-    var onCoinBaseBuyDashDialogButtonsClickListener: CoinBaseBuyDashDialogButtonsClickListener? = null
+class CoinBaseResultDialog : DialogFragment() {
```

Comment:
This class has a weird name, took me a while to find this screen.
This screen shows a "Success" or "Error" message, so "ResultDialog" seems like a more appropriate name.